BrainPortal/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20230418205141) do
+ActiveRecord::Schema.define(version: 20240408164549) do
```

Comment:
THis PR will ahev to be rebased and the version numbers edited to match the current version